components/switch/src/Switch.tsx

```diff
@@ -120,7 +120,11 @@ export const Switch = forwardRef<HTMLInputElement, SwitchProps>(
 			setChecked(isChecked);
 		}, [isChecked]);
 
-		const handleChange = () => wrapHandlers(onChange, () => setChecked(!checked));
+		const handleChange = (event: any) =>
```

Comment:
Because of these changes you will need to do a changeset file

Reply:
Yes, the changeset is what I wanted to talk with you about, when I pinged you late Fri.  I will try and chase you up in the afternoon